go/oasis-node/cmd/registry/runtime/runtime.go

```diff
@@ -405,17 +405,22 @@ func runtimeFromFlags() (*registry.Runtime, signature.Signer, error) { // nolint
 	case AdmissionPolicyNameAnyNode:
 		rt.AdmissionPolicy.AnyNode = &registry.AnyNodeRuntimeAdmissionPolicy{}
 	case AdmissionPolicyNameEntityWhitelist:
-		entities := make(map[signature.PublicKey]bool)
+		entities := make(map[signature.PublicKey]registry.EntityWhitelistConfig)
 		for _, se := range viper.GetStringSlice(CfgAdmissionPolicyEntityWhitelist) {
 			var e signature.PublicKey
 			if err = e.UnmarshalText([]byte(se)); err != nil {
 				logger.Error("failed to parse entity ID",
 					"err", err,
 					CfgAdmissionPolicyEntityWhitelist, se,
 				)
-				return nil, nil, fmt.Errorf("entity whitelist runtime admission policy parse entity ID: %w", err)
+				return nil, fmt.Errorf("entity whitelist runtime admission policy parse entity ID: %w", err)
 			}
-			entities[e] = true
+			entities[e] = registry.EntityWhitelistConfig{
+				MaxNodes: make(map[node.RolesMask]uint16),
+			}
+			// TODO: Handle the clusterfuck of parsing the nested map from
+			// command-line arguments sometime later.  As it is now, it's
+			// configured as unlimited nodes of any role for the given entity.
```

Comment:
Don't forget to file an issue.

Reply:
Filed as https://github.com/oasisprotocol/oasis-core/issues/3626 :)